pline/base.py

```diff
@@ -94,7 +94,7 @@ def iterhelper(keyvalue):
                 yield { 'key' : key, 'stringValue' : "#{%s}" % value.id }
             elif isinstance(value, bool):
                 yield { 'key' : key, 'stringValue' : str(value).lower() }
-            else:
+            elif value is not None:
```

Comment:
My reason for this is to allow effectively deleting a property from the definition by assigning None to it.  For example if earlier you assigned a schedule to a node, but then want to remove it, you could write `node.schedule = None`.